Sources/Markdown/Interpretive Nodes/Aside.swift

```diff
@@ -21,87 +21,155 @@ import Foundation
 /// > different meaning, as it doesn't quote speech.
 /// ```
 public struct Aside {
-    /// The kind of aside.
-    public enum Kind: String, CaseIterable {
+    /// Describes the different kinds of aside.
+    public struct Kind: CaseIterable, Equatable {
```

Comment:
This is an API breaking change but it makes sense to me, `Kind` shouldn't be an enum because it no longer represents a closed set. Can we call this out in the PR description?